routers/web/web.go

```diff
@@ -595,6 +595,10 @@ func registerWebRoutes(m *web.Router) {
 			m.Post("/hidden_comments", user_setting.UpdateUserHiddenComments)
 			m.Post("/theme", web.Bind(forms.UpdateThemeForm{}), user_setting.UpdateUIThemePost)
 		})
+		m.Group("/notifications", func() {
+			m.Get("", user_setting.Notifications)
+			m.Post("/email", user_setting.SetGlobalEmailNotificationPreference)
```

Comment:
It should be named as `NotificationsSetEmailPost` or `NotificationsEmailPost` , as others.